Source/gg2/Objects/Alternative Objects/2014 Haxxy and Raybanns Rewards/Tombstone.events/Create.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<event category="CREATE" id="0">
+  <actions>
+    <action id="603" library="1">
+      <!--action name: Code-->
+      <kind>CODE</kind>
+      <allowRelative>false</allowRelative>
+      <question>false</question>
+      <canApplyTo>true</canApplyTo>
+      <actionType>CODE</actionType>
+      <functionName/>
+      <relative>false</relative>
+      <not>false</not>
+      <appliesTo>.self</appliesTo>
+      <arguments>
+        <argument kind="STRING">alarm[0] = 300/global.delta_factor;
+image_speed = 1/(3 * global.ticks_per_virtual);
+ownerClass = CLASS_QUOTE; // dummy class before being set
```

Comment:
I've thought in the past we should have some mechanism to make this explicit, something like `instance_create_params`